extras/rya.geoindexing/geo.mongo/src/test/java/org/apache/rya/indexing/geotemporal/GeoTemporalProviderTest.java

```diff
@@ -36,7 +36,7 @@
 import org.openrdf.model.ValueFactory;
 import org.openrdf.model.impl.ValueFactoryImpl;
 
-public class GeoTemporalProviderTest extends GeoTemporalTestBase {
+public class GeoTemporalProviderTest extends GeoTemporalTestUtils {
```

Comment:
Do we still need to extend GeoTemporalTestUtils if all the functions are now public static?

Reply:
Nope, fixed.